### DISCUSSION TITLE
If `j` or `k` is prefixed with a count, they should ignore wrapped lines

### DISCUSSION THREAD

**Discussion by nik-rev** [helix-editor/helix#12008] [General]
Consider a file in which we have 3 lines:

- **Line 1** has 20 characters
- **Line 2** has 1000 characters
- **Line 3** has 20 characters

In Vim, If our cursor is on **Line 1**, then pressing `2j` will move us to **Line 3** by default.

But in Helix, pressing `2j` will move us to the second wrapped-line on **Line 2**. This makes relative numbers basically useless, because their main usecase is to be able to travel vertically instantly and delete large chunks of text

my proposition is that <count>k and <count>j will ignore wrapped lines, so if a line wraps the movement won't be 'absorbed'

**Comment by kirawi** [comment]
https://github.com/helix-editor/helix/wiki/FAQ#can-the-jk-bindings-be-changed-to-ignore-soft-wrapping-when-using-a-count-like-3j

**Reply by nik-rev** [reply]
Ahh thank you, `gk` and `gj` can be used to travel to absolute lines.

This is useful, but I think that `<count>j` and `<count>k` behaving like `<count>gj` and `<count>gk` also makes sense, because you wouldn't normally use a count with k or j unless you were using relative lines.

**Comment by nik-rev** [comment]
I added an updated discussion with another change I would like to see, so I'm closing this one

https://github.com/helix-editor/helix/discussions/12010
